tests/unit/ai/test_models.py

```
@@ -10,28 +10,28 @@
-        assert len(AIDecision) == 3
+        assert len(Decision) == 3
 
     def test_enum_membership(self):
         """Test that values can be checked for membership."""
-        assert AIDecision("INCLUDE") == AIDecision.INCLUDE
-        assert AIDecision("EXCLUDE") == AIDecision.EXCLUDE
-        assert AIDecision("UNCERTAIN") == AIDecision.UNCERTAIN
+        assert Decision("INCLUDE") == Decision.INCLUDE
+        assert Decision("EXCLUDE") == Decision.EXCLUDE
+        assert Decision("UNCERTAIN") == Decision.UNCERTAIN
         with pytest.raises(ValueError):
```

Ruff (0.14.7)

`pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)